ci-operator/config/openshift/installer/openshift-installer-master.yaml

```diff
@@ -304,7 +304,7 @@ tests:
   container:
     from: src
 - as: e2e-aws-ovn
-  skip_if_only_changed: (^(docs|upi|hack)/)|((^|/)OWNERS(_ALIASES)?$)|((^|/)[A-Z]+\.md$)|(^images/(agent|alibabacloud|azure|azurestack|baremetal|gcp|ibmcloud|libvirt|nutanix|openstack|ovirt|powervs|vsphere)/)
+  skip_if_only_changed: (^(docs|upi|hack)/)|((^|/)OWNERS(_ALIASES)?$)|((^|/)[A-Z]+\.md$)|(^images/(alibabacloud|azure|azurestack|baremetal|gcp|ibmcloud|libvirt|nutanix|openstack|ovirt|powervs|vsphere)|(^pkg/asset/agent/)/)
```

Comment:
Double slash at the end `^pkg/asset/agent/)/` will prevent this from working.

The only things in the images directory are `baremetal`, `installer`, `installer-artifacts`, `openstack`, `infrastructure-providers`, `installer-altinfra`, `libvirt`. So all the platform exclusion stuff here does nothing except for openstack and baremetal.

I failed to notice this either when I originally added agent to the list [here](https://github.com/openshift/release/commit/c147f3b6955978c909c9d879465eae171809fdbc#diff-92fb8761a3401342e7a5678fa916ff7f552fe64f9fa412105c354e691c76a736R203).

I assumed it was like the [others](https://github.com/openshift/release/commit/c147f3b6955978c909c9d879465eae171809fdbc#diff-92fb8761a3401342e7a5678fa916ff7f552fe64f9fa412105c354e691c76a736R233), which were of the form:

```
(^(data/.*|images|pkg(/.+)?|cmd/openshift-install|terraform/providers)/(agent|alibabacloud|aws|azure|azurestack|baremetal|ibmcloud|libvirt|nutanix|openstack|ovirt|powervs|vsphere)/)
```

The [original commit](https://github.com/openshift/release/commit/1a43860ca2ae50e527689a0b9a6345052484ed8a) was either wrong or only cared about disabling openstack.

So I think the thing we intended to have here was:

```suggestion
  skip_if_only_changed: (^(docs|upi|hack)/)|((^|/)OWNERS(_ALIASES)?$)|((^|/)[A-Z]+\.md$)|(^(data/.*|images|pkg(/.+)?|cmd/openshift-install|terraform/providers)/(agent/alibabacloud|azure|azurestack|baremetal|gcp|ibmcloud|libvirt|nutanix|openstack|ovirt|powervs|vsphere)/)
```